docs/testing.md

```diff
@@ -42,9 +42,9 @@ To run just a subset of them, specify a particular file or directory:
     py.test tests/unit
     py.test tests/unit/api.py
 
-To run just a pep8 test:
+To run just a pep8/pycodestyle test:
```

Comment:
Why not just pycodestyle?

Reply:
Yeah, we should just replace the name. Let's rename the files too (so pep8_tests.sh -> pycodestyle_tests.sh; not sure if there are any others).